src/common/impl/reporter.ts

```
@@ -25,14 +25,15 @@ export class Reporter implements IReporter {
         case 'identify':
           //Avoid identifying the user several times, until some data has changed.
           const hash = sha1(payloadString);
-          const cached = await this.cacheService?.get('identify');
+          const identifyCacheName = this.getIdentifyCacheName(this.writeKey);
```

Biome (2.5.3)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)